src/main/scala/mesosphere/marathon/api/v2/Validation.scala

```diff
@@ -91,6 +93,25 @@ object Validation {
     }
   }
 
+  def fetchUriHasSupportedProtocol: Validator[FetchUri] = {
+    lazy val supportedProtocols = Set("http", "https", "ftp", "ftps", "hdfs")
+
+    new Validator[FetchUri] {
+      def apply(uri: FetchUri) = {
+        Try {
+          val value = new URL(uri.uri)
```

Comment:
Currently mesos supports three kinds of fetcher: curl, docker and hadoop (https://github.com/apache/mesos/blob/cb1d7be772d2b4864cde3620c85e67bc4330f638/src/uri/fetcher.hpp)

I don't fully understand why simple file uris should be supported if they are not supported by mesos